widgets/RedshiftToS3-action.json

```diff
@@ -0,0 +1,136 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "DynamoDB Configuration",
+      "properties": [
+        {
+          "widget-type": "password",
+          "label": "Access Key",
+          "name": "accessKey"
+        },
+        {
+          "widget-type": "password",
+          "label": "Secret Access Key",
+          "name": "secretAccessKey"
+        },
+        {
+          "widget-type": "password",
```

Comment:
The IAM Role is kept as a password widget for authentication and security reasons. 
Please let me know if this needs to be changed to a normal text-box.